src/core/icons/pinterest.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+export const SvgPinterest = (): JSX.Element => (
+	<svg
+		width="30"
+		height="30"
+		viewBox="0 0 30 30"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
```

Comment:
Could you remove the `width`, `height` and `fill` attributes? These should be set in CSS